src/ucp/core/ucp_context.c

```diff
@@ -40,9 +40,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "In addition it's possible to use a combination of the following aliases:\n"
    " - all    : use all the available transports.\n"
    " - sm/shm : all shared memory transports.\n"
+   " - mm     : shared memory transports - only memory mappers.\n"
    " - ugni   : ugni_rdma and ugni_udt.\n"
-   " - rc     : rc and cm.\n"
-   " - rc-x   : rc with accelerated verbs and cm.\n"
+   " - rc     : rc and ud.\n"
+   " - rc-x   : rc with accelerated verbs and ud.\n"
```

Comment:
maybe we should emphasize that ud in this context selected implicitly and used for OOB.